### PR TITLE
Add Turkish translation

### PR DESCRIPTION
**Pull request by yakushabb**
Also fix some small issues mostly related to translation.

### CODE REVIEW

**Comment by yakushabb** [comment]
@josephmawa ready for review.

**Comment by josephmawa** [comment]
> @josephmawa ready for review.

Thanks for the PR. I will review as soon as I get time.

File: data/io.github.josephmawa.Bella.metainfo.xml.in
Comment (josephmawa) [inline data/io.github.josephmawa.Bella.metainfo.xml.in:1]: Thanks for this. I hope the keywords from the `.desktop` file will work as well as these keywords.

File: po/Bella.pot
Comment (josephmawa) [inline po/Bella.pot:1]: You will need to regenerate this because I've update the `POTFILES`.

File: src/application.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ export const BellaApplication = GObject.registerClass(
       });
       this.add_action(quitAction);
       this.set_accels_for_action("app.quit", ["<primary>q"]);
+      this.set_accels_for_action("app.preferences", ["<primary>comma"]);
```
Comment (josephmawa) [inline src/application.js:43]: Any particular reason why you went for this?

**Review by josephmawa — CHANGES_REQUESTED** [review]
Hi @yakushabb,  thanks for the PR. To make the translations work, I've pushed a couple of changes to main. You just need to update your feature branch and resolve the merge conflict. It should be good to merge.

File: data/io.github.josephmawa.Bella.metainfo.xml.in
Comment (yakushabb) [reply data/io.github.josephmawa.Bella.metainfo.xml.in:1]: "If there’s a [launchable](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#launchable) defined for a desktop application, categories and keywords are pulled from the desktop file. Defining them separately in the Metainfo file will override the contents of the desktop file."

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#categories-and-keywords

File: src/application.js (hunk 1 above)
Comment (yakushabb) [reply src/application.js]: Standard Keyboard Shortcuts

Preferences Ctrl+, Opens the preferences window

https://developer.gnome.org/hig/reference/keyboard.html

File: src/window.js [hunk 2]
```diff
@@ -248,7 +248,6 @@ export const BellaWindow = GObject.registerClass(
 
           if (color) {
             this.copyToClipboard(color);
-            // Translators: Do NOT translate %s. It is a placeholder.
```
[the remark this answers is not in the document]
Comment (yakushabb) [reply src/window.js]: Definitely unnecessary.

**Comment by josephmawa** [comment]
Thanks for the PR. I hope you will check here once in a while to check for translation updates. There are several color formats I'm yet to add.

**Pull request merged**